Calling attach method of transform controls generates type errors

##### Description of the problem
I am using three version 105 in my angular app
On compiling the project. I get the following type error in TransformControls.d.ts

```
ERROR in node_modules/three/examples/jsm/controls/TransformControls.d.ts(31,3): error TS2416: Property 'attach' in type 'TransformControls' is not assignable to the same property in base type 'Object3D'.
  Type '(object: Object3D) => void' is not assignable to type '(object: Object3D) => this'.
    Type 'void' is not assignable to type 'this'.
```
And following is the method where I call attach method

Note: tControls is an instance of TranformControls
```
importSolarPanel(){
    const loader = new GLTFLoader()
    loader.load('assets/scene.gltf', (gltf)=>{
      console.log(gltf)
      gltf.scene.traverse(mesh=>{
        if(mesh instanceof Mesh){
          this.tControls.attach(mesh)
        }
      })
      this.tControls.setMode('rotate')
      this.scene.add(gltf.scene)
    }, (ev)=>{
      console.log(ev)
    }, (err)=>{
      console.log(err)
    })
  }
```

##### Three.js version

- [ ] Dev
- [x] r105
- [ ] ...

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

##### Hardware Requirements (graphics card, VR Device, ...)

--- a/examples/js/controls/TransformControls.js
+++ b/examples/js/controls/TransformControls.js
@@ -142,6 +142,8 @@ THREE.TransformControls = function ( camera, domElement ) {
 		this.object = object;
 		this.visible = true;
 
+		return this;
+
 	};
 
 	// Detatch from object
@@ -150,6 +152,8 @@ THREE.TransformControls = function ( camera, domElement ) {
 		this.object = undefined;
 		this.visible = false;
 		this.axis = null;
+
+		return this;
 
 	};
 

--- a/examples/jsm/controls/TransformControls.d.ts
+++ b/examples/jsm/controls/TransformControls.d.ts
@@ -29,7 +29,7 @@ export class TransformControls extends Object3D {
   visible: boolean;
 
   attach(object: Object3D): this;
-  detach(): void;
+  detach(): this;
   pointerHover(pointer: Object): void;
   pointerDown(pointer: Object): void;
   pointerMove(pointer: Object): void;

--- a/examples/jsm/controls/TransformControls.js
+++ b/examples/jsm/controls/TransformControls.js
@@ -167,6 +167,8 @@ var TransformControls = function ( camera, domElement ) {
 		this.object = object;
 		this.visible = true;
 
+		return this;
+
 	};
 
 	// Detatch from object
@@ -175,6 +177,8 @@ var TransformControls = function ( camera, domElement ) {
 		this.object = undefined;
 		this.visible = false;
 		this.axis = null;
+
+		return this;
 
 	};
 

